tools/cli/go-whisk-cli/commands/sdk.go

```diff
@@ -74,17 +75,23 @@ var sdkInstallCmd = &cobra.Command{
         case "ios":
             err = iOSInstall()
         case "bashauto":
-            err = WskCmd.GenBashCompletionFile(BASH_AUTOCOMPLETE_FILENAME)
-            if (err != nil) {
-                whisk.Debug(whisk.DbgError, "GenBashCompletionFile('%s`) error: \n", BASH_AUTOCOMPLETE_FILENAME, err)
-                errStr := wski18n.T("Unable to generate '{{.name}}': {{.err}}",
-                        map[string]interface{}{"name": BASH_AUTOCOMPLETE_FILENAME, "err": err})
+            whisk.Debug(whisk.DbgInfo, "Running bashauto script\n")
+            if err = WskCmd.GenBashCompletion(os.Stdout); err != nil {
+                whisk.Debug(whisk.DbgError, "GenBashCompletion error: %s\n", err)
+                errStr := wski18n.T("Unable to generate bashauto-completion {{.err}}",
```

Comment:
trying to maintain consistent wording with docs.... `unable to output...` (use `output` instead of `generate`)